.changeset/calm-spies-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@layerzerolabs/ua-devtools-evm-hardhat": patch
```

Comment:
I think we are missing devtools from this changeset

Reply:
Aaaand fixed